python-frontend/src/main/java/org/sonar/python/lexer/FStringChannel.java

```diff
@@ -154,7 +157,7 @@ private static boolean canConsumeFStringPrefix(StringBuilder sb, CodeReader code
   }
 
   private static boolean isRawStringBackSlash(CodeReader code, FStringState state) {
```

Comment:
Maybe for future us and for clarity we could be super explicit on the function name to only target single backslashes.
```suggestion
  private static boolean isRawStringSingleBackSlash(CodeReader code, FStringState state) {
```